### PR TITLE
Apple Clone is added

### LINKED ISSUE #2231: Apple-Clone
**Describe the project you want to add with tech stack**

Apple-Clone with  html, Css 

Assign this issue to me under GSSOC'23

### PR DESCRIPTION
**Pull request by shraddha761**
# Title and Issue number 

Close #2231 

# Checklist:

- [x] I have mentioned the issue number in my Pull Request.
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have created a helpful and easy to understand `README.md`

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -759,6 +759,10 @@ Happy Coding! -->
 
 <tr align="center">
 <td align="center"> 250.</td>
+<td align="center"> <a href="https://github.com/pranjay-poddar/Dev-Geeks/tree/main/Frontend-Projects/ApppleClone"> Apple Clone </a></td>
```
Comment (TusharKesarwani) [inline README.md:762]: Create new project item
Do not add it in one item

Comment (shraddha761) [reply README.md:762]: > 

Sorry will you please elaborate

Comment (TusharKesarwani) [reply README.md:762]: Like create new item for your project addition
Do not add new project in already added project item

**Comment by shraddha761** [comment]
@TusharKesarwani please review

File: README.md [hunk 2]
```diff
@@ -756,6 +756,8 @@ Happy Coding! -->
 </tr>
 
 <tr align="center">
+<td align="center"> <a href="https://github.com/pranjay-poddar/Dev-Geeks/tree/main/Frontend-Projects/QrComponent"> QrComponent </a></td>
```
Comment (TusharKesarwani) [inline README.md:759]: Why are you adding these projects?

Comment (shraddha761) [reply README.md:759]: I am not adding these projects I don't know why it is happening

**Comment by shraddha761** [comment]
@TusharKesarwani Is it fine now

File: README.md (hunk 2 above)
Comment (TusharKesarwani) [reply README.md:759]: Remove all the changes and add your project

**Pull request closed without merging**